packages/form/src/props.ts

```diff
@@ -1,28 +1,52 @@
-import { TextInputProps } from '@openstad-headless/ui/src/form-elements/text/props';
+import { TextInputProps } from '@openstad-headless/ui/src/form-elements/text';
 import { TickmarkSliderProps } from "@openstad-headless/ui/src/form-elements/tickmark-slider";
 import { RangeSliderProps } from "@openstad-headless/ui/src/form-elements/a-b-slider";
 import { CheckboxFieldProps } from "@openstad-headless/ui/src/form-elements/checkbox";
 import { SelectFieldProps } from "@openstad-headless/ui/src/form-elements/select";
 import { RadioboxFieldProps } from "@openstad-headless/ui/src/form-elements/radio";
 import { FileUploadProps} from "@openstad-headless/ui/src/form-elements/file-upload";
+import { HiddenInputProps } from "@openstad-headless/ui/src/form-elements/hidden";
+import {ImageChoiceFieldProps} from "@openstad-headless/ui/src/form-elements/image-choice";
 
 export type FormProps = {
     title?: string;
     fields: CombinedFieldProps[];
     submitText?: string;
     submitHandler: (data: any) => void;
+    submitDisabled?: boolean;
     saveAsConceptLabel?: string;
```

Comment:
Kunnen we dit voor semantics secondaryHandler sen scondarylabel noemen? Op die manier is de hint beter dat je een secondaire actie kan toevoegen en deze knop niet alleen voor de resourceform te gebruiken is

Reply:
bijv voor een cancel of andere situatie